shared/js/background/redirect.es6.js

```diff
@@ -164,7 +166,7 @@ function handleRequest (requestData) {
             }
             browserWrapper.notifyPopup({ updateTabData: true })
             // Block the request if the site is not allowlisted
-            if (blockingEnabled && !thisTab.site.isAllowlisted() && tracker.action.match(/block|redirect/)) {
+            if (blockingEnabled && tracker.action.match(/block|redirect/)) {
```

Comment:
Removed allowlist because `blockingEnabled` is now doing the correct checks of both broken lists.